src/extension-test/unit/util/cursor-get-text-test.ts

```diff
@@ -3,64 +3,64 @@ import { docFromTextNotation } from '../common/text-notation';
 import * as getText from '../../../util/cursor-get-text';
 
 describe('get text', () => {
-    describe('getTopLevelFunction', () => {
-        it('Finds top level function at top', () => {
-            const a = docFromTextNotation(
-                '(foo bar)•(deftest a-test•  (baz |gaz))'
-            );
-            const b = docFromTextNotation(
-                '(foo bar)•(deftest |a-test|•  (baz gaz))'
-            );
-            const range: [number, number] = [b.selectionLeft, b.selectionRight];
-            expect(getText.currentTopLevelFunction(a)).toEqual([
-                range,
-                b.model.getText(...range),
-            ]);
-        });
+    // describe('getTopLevelFunction', () => {
+    //     it('Finds top level function at top', () => {
+    //         const a = docFromTextNotation(
+    //             '(foo bar)•(deftest a-test•  (baz |gaz))'
+    //         );
+    //         const b = docFromTextNotation(
+    //             '(foo bar)•(deftest |a-test|•  (baz gaz))'
+    //         );
+    //         const range: [number, number] = [b.selectionLeft, b.selectionRight];
+    //         expect(getText.currentTopLevelFunction(a)).toEqual([
+    //             range,
+    //             b.model.getText(...range),
+    //         ]);
+    //     });
```

Comment:
Instead of commenting, you can use `it.skip` (possibly also `describe.skip`, idk). There's a report on the number of skipped tests in at the end of the test output.

Reply:
Yes `skip` doesn't signal ”old dead code that should be cleaned out” 😄 Apart from also being tallied and reported. We currently have 4 pending tests the report says:

<img width="955" alt="image" src="https://user-images.githubusercontent.com/30010/157071744-0068b381-e4f9-4666-9160-d1e9c5a350e0.png">

And I do intend to get back to them. Haha.